src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,48 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private final String lineSeparator = System.lineSeparator();
```

Comment:
The variable 'lineSeparator' should be declared as a constant. It's a common practice in Java to declare constants using the 'static final' modifiers, and their names should be in uppercase. For example: private static final String LINE_SEPARATOR = System.lineSeparator();